Operators/Types/user/Dom/mapping/MapImageToResolution_a5de95e9-008c-4a26-b9f5-65e842d16d74.t3ui

```diff
@@ -0,0 +1,212 @@
+{
+  "Id": "a5de95e9-008c-4a26-b9f5-65e842d16d74"/*MapImageToResolution*/,
+  "Description": "Maps the provided image to the provided resolution.\n\nIf any dimension is left at the default (0), then it will simply map to the context's [RequestedResolution]",
+  "InputUis": [
```

Comment:
Would it make sense to call this op [SetImageResolution] ?

Reply:
it might, i dont have any strong feelings about it either way!